ihatemoney/tests/budget_test.py

```diff
@@ -119,6 +119,7 @@ def test_invite(self):
         resp = self.client.get("/raclette/join/token.invalid", follow_redirects=True)
         self.assertIn("Provided token is invalid", resp.data.decode("utf-8"))
 
+    @pytest.mark.usefixtures("app_ctx")
```

Comment:
Is there a way to avoid repeating this line every so often?

Reply:
I would recommend having a look at [pytest-flask](https://pypi.org/project/pytest-flask/). I think this is not documented, but this allows among other things to access the flask app context directly in the tests (without `with app.app_context(): ...`